### PR TITLE
Add hsimple copying test rule in file merger

### PR DESCRIPTION
**Pull request by linev**
1. When running tests, one cannot use dependency from normal targets. 
Therefore, in `filemerger` create/copy hsimple.root file using extra tests targets and assign dependency from them

2. `TIMEOUT` parameter in ROOTTEST_ADD_TEST can only have single value

3. When generate dictionary, use name of dictionary directly - not a global variable which may change in the future

### CODE REVIEW

**Comment by oshadura** [comment]
@phsft-bot build with flags -DCTEST_TEST_EXCLUDE_NONE=On

**Comment by etejedor** [comment]
Please don't merge yet, I need to check the issue of the temporaries in the src directory reported by @linev

File: python/JupyROOT/CMakeLists.txt [hunk 1]
```diff
@@ -1,17 +1,18 @@
 if (ROOT_pyroot_FOUND AND NOT ROOT_CLASSIC_BUILD) # Do not run with classic build
 
 set(MODULES_LOCATION ${ROOTSYS}/lib/JupyROOT/helpers)
-set(NBDIFFUTIL ${CMAKE_CURRENT_SOURCE_DIR}/nbdiff.py )
+set(NBDIFFUTIL ${CMAKE_CURRENT_SOURCE_DIR}/nbdiff.py)
```
Comment (etejedor) [inline python/JupyROOT/CMakeLists.txt:4]: Changes in this file are superseded by:

https://github.com/root-project/roottest/pull/511

Please remove this commit from the PR, thanks @linev !

**Comment by linev** [comment]
@etejedor, I removed JupyROOT part from PR

**Comment by etejedor** [comment]
> @etejedor, I removed JupyROOT part from PR

Ok thanks, then I leave up to @oshadura to decide about the rest (and she actually already approved, so should be ok to merge).

**Pull request merged**